client/src/components/podcasts/PodcastShowPage.js

```diff
@@ -23,6 +23,8 @@ const PodcastShowPage = (props) => {
       }
       const body = await response.json()
       setPodcast(body.podcast)
+      setReviews(body.podcast.reviews)
```

Comment:
Here we wouldn't need to set 2 separate pieces of state, we can just `setPodcast(body.podcast)` since the `body.podcast` contains `reviews`